test_support/src/lib.rs

```diff
@@ -6,6 +6,7 @@ use std::fmt::Formatter;
 use std::time::Duration;
 
 const PORT: u16 = 8080;
+const TIMEOUT: u64 = 10;
```

Comment:
I presume this is because previously the test was already borderline not completing within the timeout? In which case bumping seems worth doing.

Though it seems strange that the timeout now needs bumping, when this PR means the implicit case now should be faster, not slower? Unless just bad luck with CI being slower today than normal etc.

Reply:
yes, the integration tests would always timeout intermittently for me locally.  just tweaking this because it seems to give more stability when running them locally and matches the timeout [chosen for the builder workflow](https://github.com/heroku/builder/blob/main/.github/workflows/build-test-publish.yml#L75)